rest/data/admin.go

```diff
@@ -24,6 +24,20 @@ func GetBanner(ctx context.Context) (string, string, error) {
 	return settings.Banner, string(settings.BannerTheme), nil
 }
 
+func GetServiceFlags(ctx context.Context) (evergreen.ServiceFlags, error) {
+	settings, err := evergreen.GetConfig(ctx)
```

Comment:
Can we instead use `evergreen.GetServiceFlags`, filter out the result and rename this to something like `GetNecessaryServiceFlags`? To avoid having two similar `GetServiceFlags` functions